src/screens/ENSSearchSheet.js

```diff
@@ -135,14 +135,7 @@ export default function ENSSearchSheet() {
               </Text>
             </Inline>
           )}
-          {isIdle && (
-            <>
-              <Inset vertical="24px">
-                <Divider />
-              </Inset>
-              <PendingRegistrations />
-            </>
-          )}
+          {isIdle && <PendingRegistrations />}
```

Comment:
`isIdle` didn't make it? we talked with @jxom about this when building this feature and it was working

Reply:
No, I can see if I can tweak it to make isIdle do the trick